acceptance/ui/features/api/cli_service.rb

```diff
@@ -148,6 +161,27 @@ def check_publicendpoint_port_exists_in_service?(service, endpoint, portAddr, pr
             return false
         end
 
+        # Looks up the given vhost from the vhosts table, and returns the enabled
+        # state based on the service definition.
+        def check_publicendpoint_vhost_enabled_in_service_json?(name)
```

Comment:
I started to do this, but the two have slightly different data members.  The service check is checking for the existence of different pieces of data between the two of them and the json data has different keys/number of values.